packages/fxa-js-client/tests/lib/account.js

```diff
@@ -2,613 +2,607 @@
-              }),
-              RequestMocks.signIn
-            );
-          })
-          .then(function(result) {
-            assert.ok(result.uid);
-          }, assert.notOk);
-      });
+  it('#accountStatus', function() {
+    return accountHelper
+      .newVerifiedAccount()
+      .then(function(result) {
+        return respond(
+          client.accountStatus(result.signIn.uid),
+          RequestMocks.accountStatus
+        );
+      })
+      .then(function(res) {
+        assert.equal(res.exists, true);
+      }, assert.notOk);
+  });
+
+  it('#accountProfile', function() {
+    return accountHelper
+      .newVerifiedAccount()
+      .then(function(account) {
+        return respond(
+          client.accountProfile(account.signIn.sessionToken),
+          RequestMocks.accountProfile
+        );
+      })
+      .then(function(res) {
+        assert.isNotNull(res);
+      }, assert.notOk);
+  });
+
+  it('#accountStatus with wrong uid', function() {
+    return respond(
+      client.accountStatus('00047f01e387498e8ccc7fede1a74000'),
+      RequestMocks.accountStatusFalse
+    ).then(function(res) {
+      assert.equal(res.exists, false);
+    }, assert.notOk);
+  });
+
+  it('#accountStatus with no uid', function() {
+    return client.accountStatus().then(
+      function() {
+        assert.fail('client.accountStatus should reject if uid is missing');
+      },
+      function(err) {
+        assert.equal(err.message, 'Missing uid');
+      }
+    );
+  });
+
+  it('#accountStatusByEmail', function() {
+    return accountHelper
+      .newVerifiedAccount()
+      .then(function(result) {
+        return respond(
+          client.accountStatusByEmail(result.input.email),
+          RequestMocks.accountStatus
+        );
+      })
+      .then(function(res) {
+        assert.equal(res.exists, true);
+      }, assert.notOk);
+  });
+
+  it('#accountStatusByEmail with wrong email', function() {
+    return respond(
+      client.accountStatusByEmail('invalid@email.com'),
+      RequestMocks.accountStatusFalse
+    ).then(function(res) {
+      assert.equal(res.exists, false);
+    }, assert.notOk);
+  });
+
+  it('#accountStatusByEmail with no email', function() {
+    return client.accountStatusByEmail().then(
+      function() {
+        assert.fail(
+          'client.accountStatusByEmail should reject if email is missing'
+        );
+      },
+      function(err) {
+        assert.equal(err.message, 'Missing email');
+      }
+    );
+  });
+
+  it('#login unblock accept', function() {
+    var user = 'block.' + new Date().getTime();
+    var email = user + '@restmail.net';
+    var password = 'iliketurtles';
+
+    return respond(client.signUp(email, password), RequestMocks.signUp)
+      .then(function(result) {
+        return respond(
+          client.signIn(email, password, { context: 'fx_desktop_v3' }),
+          ErrorMocks.signInBlocked
+        );
+      })
+      .then(assert.notOk, function(error) {
```

Comment:
This is curious. `assert.notOk` is not supposed to be executed is it, so did it need to change? Are the assertions below still being executed?